requirements.txt

```diff
@@ -1,4 +1,4 @@
-    pytket[ZX] == 1.16.0rc0
+    pytket[ZX] == 1.16.0
     pytket-projectq == 0.27.0
     pytket-qiskit == 0.39.0
     pytket-qsharp == 0.33.0
```

Comment:
It may be worth waiting for new releases of pytket-qiskit and pytket-qsharp and updating the versions here, but otherwise LGTM apart from one typo.

Reply:
I would assume that pytket-qiskit will take a few more days, maybe not worth waiting. But Maybe we should think about pytket-cirq and pytket-pyquil?

Reply:
(If you want to continue now, I am happy to update the extensions later after they have been released)

Reply:
pytket-qsharp, pytket-pyquil and pytket-projectq do not appear in the manual. This is a shared requirements file for the examples repository.

IMO we should separate the requirements into two separate files so its clearer which are used by the manual and which ones are used by the examples repo.

I'm in no rush to merge this so happy to wait for the pytket-qiskit and pytket-cirq releases

Reply:
At the moment they are not, but cirq and pyquil should be in there, they are not used at the moment because of the networkx issue, see: https://github.com/CQCL/pytket/commit/2c6ce3b584c2dd202eb982a03be80d6f39538144

I think You should merge this now and we should do the update and reactivation of them later